spec/queueSpec.js

```diff
@@ -0,0 +1,69 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/queue'
+
+chai.use(chaiChange)
+describe('Queue', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(Queue).to.be.a('function')
+  })
+
+  context('enqueue(element)', () => {
+    it('pushes an element to the back of the list.', () => {
+      const myQueue = new Queue()
+
+      expect(() => myQueue.enqueue('foo')).to.alter(() => myQueue.length(), { from: 0, to: 1 })
+
+      expect(() => myQueue.enqueue('boo')).to.alter(() => myQueue.length(), {from: 1, to: 2})
+
+      expect(() => myQueue.enqueue('moo')).to.alter(() => myQueue.length(), {from: 2, to: 3})
```

Comment:
refactor enqueue test to make sure 1. that specific element is in the queue, 2. that specific element is in the right place